docs/examples/taylor.ipynb

```diff
@@ -0,0 +1,120 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `set_titles_and_labels`\n",
+    "\n",
+    "This notebook is a simple example of the GeoCAT-viz function <a href=\"../user_api/generated/geocat.viz.util.set_titles_and_labels.html#geocat-viz.util.set_titles_and_labels\">set_titles_and_labels</a>."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import packages:\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import xarray as xr\n",
+    "\n",
+    "import cftime\n",
```

Comment:
The example seems to be failing on the cftime import (though not loudly - which is concerning).
https://geocat-viz--186.org.readthedocs.build/en/186/examples/taylor.html

Do we need this or is it something we could strip out?